recipes-app/iot2050-firmware-update/files/iot2050-firmware-update

```diff
@@ -461,14 +458,21 @@ def main(argv):
     else:
         updater = ManagedFirmwareUpdate(args.firmware)
 
-    envlist = updater.get_preserved_uboot_env(args.preserve_list)
+    envlist = None
+    if not args.force:
+        envlist = updater.get_preserved_uboot_env(args.preserve_list)
+
     updater.update_firmware()
+
     if envlist:
         print("\nRestore preserved uboot env")
         updater.update_uboot_env(envlist)
 
-    print("\nCompleted. Please reboot the device\n")
-
+    reboot_input = input(
+        "\nWarning: Completed. Please reboot the device. Continue (y/N)? ")
```

Comment:
`Continue` sounds to me too vague. More clear to me is to use `Reboot now (y/N)?`

Reply:
This prompt is better, it has been modified